marklogic-data-hub-central/ui-custom/src/components/DateRangeFacet/DateRangeFacet.scss

```diff
@@ -0,0 +1,79 @@
+.title {
```

Comment:
Since this title CSS rule is not scoped inside a class name for the widget, it is getting applied to all the instances of class name in "title" in the application. There are a lot of "title" instances. For example in the header ("Entity Viewer") and the Metrics widget ("New entities this week").

I think we can actually remove the CSS .title definition here (and the enclosed .facetInfo) since this is already defined inside the Facets.scss file.

Reply:
Done!!